docs/autoplay.md

```diff
@@ -0,0 +1,82 @@
+## Managing Auto Play
+If you would like to start automatically playing your content, you can easily manipulating the player configuration and set the `playback.autoplay` value accordingly.
```

Comment:
If you would to start playing your content automatically, simply set the value of the Player configuration 'playback.autoplay' accordingly.